README.md

```diff
@@ -181,6 +181,43 @@ NAME                            READY   STATUS    RESTARTS   AGE
 policy-local-59854877c9-xwtfk   1/1     Running   0          2m38s
 ```
 
+### Podman Support
+
+Podman does not allow binding to privileged ports (e.g., 1-1024) unless it's
```

Comment:
```suggestion
When running cloud-provider-kind in a container on Windows or macOS, accessing LoadBalancer services can be challenging. Similar problem happens when not running Podman as root , since does not allow binding to privileged ports (e.g., 1-1024). The -enable-lb-port-mapping flag provides a solution by enabling the necessary port mapping, allowing host access to these services.
```